README.rst

```diff
@@ -1279,6 +1279,30 @@ finds one that no longer finds a ``redirect`` directive.  At that point it will
 ``buildrunner.yaml``.  It will **not** merge entries from earlier, redirecting ``buildrunner.yaml``
 files - all other content is ignored when the ``redirect`` directive is found.
 
+Cleaning Cache
+==============
+
+Buildrunner keeps a local cache in the ``~/.buildrunner/caches`` directory that will grow over time and
```

Comment:
The default location is currently `~/.buildrunner/caches` but it could be different in the config and be saved elsewhere.

```
  # The 'caches-root' global configuration specifies the directory to use for
  # build caches. The default directory is ~/.buildrunner/caches.
  caches-root: ~/.buildrunner/caches
```

Reply:
Fair point...